app/src/data/schemas/collectionSchema.ts

```diff
@@ -0,0 +1,14 @@
+const collectionObject = {
```

Comment:
Where is this being used? Or is it for reference?

Reply:
just for reference